com.unity.render-pipelines.universal/Runtime/UniversalRenderer.cs

```diff
@@ -519,6 +519,12 @@ public override void Setup(ScriptableRenderContext context, ref RenderingData re
                 createColorTexture = createDepthTexture;
             }
 
+            bool requiresDepthCopyPass = !requiresDepthPrepass
+                && requiresDepthTexture
+                && createDepthTexture;
+            // Post process passes that require depth texture (Motion Blur / DoF) as shader read resource will need to access depth information from main geometry pass if there is no depth copy
+            // and depth pre-pass enqueued
+            bool postProcessingUseDepthTextureMainPass = !requiresDepthCopyPass && !requiresDepthPrepass && cameraHasPostProcessingWithDepth;
```

Comment:
```suggestion
            bool postProcessingUseDepthTextureMainPass = !requiresDepthCopyPass && cameraHasPostProcessingWithDepth;
```
 
 `requiresDepthPrepass` is reduntant logic here. If you expand `requiresDepthCopyPass` the expression you get:

bool postProcessingUseDepthTExtureMainPass = (!**requiresDepthPrepass** && requiresDepthTexture && createDepthTexture) && !**requiresDepthPrepass** && cameraHasPostProcessingWithDepth;